android/test_app/app/build.gradle

```diff
@@ -101,8 +102,8 @@ dependencies {
     implementation 'com.facebook.soloader:nativeloader:0.8.0'
     localImplementation project(':ops')
 
-    implementation 'org.pytorch:pytorch_android:1.8.0-SNAPSHOT'
-    implementation 'org.pytorch:pytorch_android_torchvision:1.8.0-SNAPSHOT'
+    implementation "org.pytorch:pytorch_android:$pytorchAndroidVersion"
+    implementation "org.pytorch:pytorch_android_torchvision:$pytorchAndroidVersion"
```

Comment:
I think we should discuss if we would want torchvision android artifacts to be versioned with the same version as PyTorch or with the version of torchvision (which is different than Pytorch versions).

In one hand, having the same version as PyTorch makes it easier for matching dependencies, on the other hand there are discrepancies wrt what we ship in the Python side. Thoughts?